spec/javascripts/controllers/host/host_form_controller_spec.js

```diff
@@ -214,11 +214,11 @@ describe('hostFormController', function() {
 
     it('returns true if password fields are left blank', function() {
```

Comment:
Small change request - update comment to match the return "false'

Reply:
@lgalis fixed comment